src/chains/opStack/utils/getWithdrawals.ts

```diff
@@ -0,0 +1,20 @@
+import type { ErrorType } from '../../../errors/utils.js'
+import type { Log } from '../../../types/log.js'
+import type { Withdrawal } from '../types/withdrawal.js'
+import { extractWithdrawalMessageLogs } from './extractWithdrawalMessageLogs.js'
+
+export type GetWithdrawalsParameters = {
+  /** The L2 transaction receipt logs. */
+  logs: Log[]
+}
+
+export type GetWithdrawalsReturnType = Withdrawal[]
+
+export type GetWithdrawalsErrorType = ErrorType
```

Comment:
```suggestion
export type GetWithdrawalsErrorType = ErrorType | ExtractWithdrawalMessageLogsErrorType
```
Should this include the underlying error type?

Reply:
ah yes